__tests__/server.test.js

```diff
@@ -146,6 +146,40 @@ describe("backend API project", () => {
           });
         });
     });
+    test("that the articles are ordered by votes in  desc order when given a sort_by of votes and no order query", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.articles)).toBe(true);
+          expect(body.articles.length).toBeGreaterThan(0);
+          expect(body.articles).toBeSortedBy("votes", { descending: true });
+        });
+    });
+    test("that the articles are ordered by titles and in asc order when given a sort_by of titles and a order of asc", () => {
+      return request(app)
+        .get("/api/articles?sort_by=title&order=asc")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeSortedBy("title");
+        });
+    });
```

Comment:
Nice, but we should first be testing that order by can be queried on its own, so /api/articles?order=asc should also work and use the default created_at column successfully as well.